internal/condition/user_condition_test.go

```diff
@@ -0,0 +1,30 @@
+package condition
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/testutils/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_get_proxy_response(t *testing.T) {
+	// TODO(chmouel): Need to get the actual response from the proxy, built this
```

Comment:
what do you mean here?

Reply:
I just built this out of the structs i haven't tested what we actually get,

Reply:
ok